Boolean.CSharp.Main/Customer.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Boolean.CSharp.Main
+{
+    public class Customer
+    {
+        private List<Account> accounts;
+        private string name;
+        public Customer(string name) 
+        {
+            this.name = name;
+            accounts = new List<Account>();
+        }
+
+        public bool createAccount(Account acc)
+        {
+            bool result = false;
+
+            if (accounts.Contains(acc) == false){
+                accounts.Add(acc);
+                result = true;
+            }
+
+            return result;
+        }
+
+        public bool depositAccount(string accName, float amount, string date)
```

Comment:
not sure why you have a deposit method on the Bank, the Customer and the Account; i'd expect it only on the Account. The flow should be:

i go to bank, get the Account, then with the Account i do deposit / withdraw